data/korath/korath missions.txt

```diff
@@ -257,3 +257,223 @@ mission "Small Scale Delivery"
+		
+		"Gaktem GP Hybrid Thruster"
+		"Gaktem GP Hybrid Steering"
+		"Scram Drive"
+
+mission "Efreti: Distress Call 4"
+	name "Salvage ship"
+	description "Salvage the Korath ship."
+	source "Greenwater"
+	stopover "Seleptra Nak"
+	destination "Karek Fornati"
+	landing
+	passengers 66
+	blocked "You need to be able to carry <bunks> additional crew to salvage the ship. Return here when you have sufficient crew capacity."
+
+	npc
+		system "Seketra"
+		fleet "Small Kor Mereti" 2
+		government "Kor Mereti"
+		personality	
+			staying
+
+	to offer
+		has "Efreti: Distress Call 3: done"
+
+	on stopover
+		conversation
+			`Thanks to your engineer's work the atmosphere on the ship is to Human standards, if a bit tropical. Your crew works on repairs and getting familiar with the ship. With it's strong hull it appears spaceworthy but your engineer tells you that you have no weapons, shield regeneration or battery. If the Mereti attack, you are a sitting duck.`
+			choice
+				`	(Fly the <EDC-ship> home)`
+					goto runforit
+				`	(Abandon the ship)`
+					goto abandon
+			
+			label abandon
+			action
+				fail
+			`	Your crew abandons the Korath ship the elements. One more tragic story amid the ruins of their civilisation.`
+				flee
+			
+			label runforit
+			action
+				# Ship name should match the <EDC-ship> substitution above.
+				give ship "Charm-Shallop (Distressed)" "Esketari"
+			`	You manage to bring up the shields, at least initially, using the <ship>. Your crew on the <EDC-ship> take their stations as you prepare to evade Mereti patrols on your way to <destination>.`
+				launch
+	
+	on complete
+		conversation
+			`There are no crowds waiting for you when you land. Rather the ground crews seem astonished to see the ship. It takes you some time to arrange for repairs but at least they are familiar with the ship. When you come out you see a pavilion has been set up near the <EDC-ship> and small groups of Korath periodically stop by.`
+			choice
+				`	(Check it out.)`
+				`	(I've got places to be.)`
+					decline
+				
+			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in some kind of robes stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand. It is fermented and has a sweet and sour taste that leaves you lips and mouth tingling and then numb. They press the sweets on you so you take them. You see a few groups from the spaceport follow the same ritual, though some of them walk up to the "choir" and say something to them. When you finish the food, you take your leave and return to your ship.`
+
+
+mission "Efreti: Distress Call 5"
+	name "Translate Title"
```

Comment:
Title? What makes me think it's any sort of title?

Reply:
This is mentioned in the original offer for this chain where the Quarg translates.

Reply:
The data chip is described as reminding you of a license. This was to hint that it might be a legal document of some kind. However, this is an invisible mission, so I don't think the player will ever be exposed to any confusion from the name.